.github/workflows/nintendo.yaml

```diff
@@ -0,0 +1,40 @@
+# based on https://angband.readthedocs.io/en/latest/hacking/compiling.html#nintendo-ds-nintendo-3ds
+name: Nintendo DS
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  _3ds:
```

Comment:
Why is this named `_3ds`?

Reply:
Github Actions job name could not start with a numeric character https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_id